port/win/env_win.h

```diff
@@ -97,7 +97,7 @@ class WinClock : public SystemClock {
 
   Status GetCurrentTime(int64_t* unix_time) override;
   // Converts seconds-since-Jan-01-1970 to a printable string
-  virtual std::string TimeToString(uint64_t time);
+  virtual std::string TimeToString(uint64_t time) override;
```

Comment:
Preferred to elide `virtual` with `override`

Reply:
Done.